File: Basics/Java/Q7.java
import java.util.Scanner;

class Q7{
     public static void main(String[] args) {
         Scanner sc = new Scanner(System.in);
           System.out.println("Enter a temperature: ");
           int temp =sc.nextInt();

           if(temp > 30)System.out.println("It's very hot!");
           else if(temp > 10 && temp < 30) System.out.println("The weather is pleasant.");
           else System.out.println("It's very cold!");
     }
}

File: Basics/Dart/Q7.dart
import 'dart:io';

void main() {
  print("Enter the temperature: ");
  int temp = int.parse(stdin.readLineSync()!);

  if (temp > 30)
    print("It's very hot!");
  else if (temp > 10 && temp < 30)
    print("The weather is pleasant.");
  else
    print("It's very cold!");
}
